README.md

```diff
@@ -1,2 +1,33 @@
 # cantaloupe-auth-delegate
-A Java delegate for supporting IIIF Auth in Cantaloupe
+
+A Java delegate for supporting IIIF Auth in Cantaloupe.
+
+This is not ready for use. If you've found this already somehow, bookmark it and return again later. Thanks.
+
+### Building the Delegate
+
+There is a two step build process. First, to install the cantaloupe jar in your local Maven repository, run:
+
+    mvn validate
+
+After that is done, you can run the following to build the delegate:
+
+    mvn verify
```

Comment:
`mvn verify` failed with

```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] error reading ~/.m2/repository/edu/illinois/library/cantaloupe/cantaloupe/5.0.2/cantaloupe-5.0.2.jar; zip END header not found
[ERROR] UCLALibrary/cantaloupe-auth-delegate/src/main/java/edu/ucla/library/iiif/auth/delegate/CantaloupeAuthDelegate.java:[2,1] cannot access edu.ucla.library.iiif.auth.delegate
  zip END header not found
[INFO] 2 errors
```

My setup:
```
$ git status
On branch IIIF-1113
Your branch is up to date with 'origin/IIIF-1113'.

nothing to commit, working tree clean

$ git show --oneline | head -n 1
cffafcc Remove eclipse configs

$ mvn --version
Apache Maven 3.6.0
Maven home: /usr/share/maven
Java version: 11.0.11, vendor: Ubuntu, runtime: /usr/lib/jvm/java-11-openjdk-amd64
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "5.3.0-7648-generic", arch: "amd64", family: "unix"
```

Reply:
Hmm, at the last minute I used Git LFS because it warned me the cantaloupe jar was too large to check into GitHub directly. I'm not super familiar with how that works though. It may be that you need it installed on your machine too. Would you do that and try again?

Reply:
That did it 😀 I used the version of `git-lfs` in the main Ubuntu bionic repo (2.3.4-1)

Reply:
I'll update the README with that requirement.